node/basic-cloud-functions-bot/index.js

```diff
@@ -18,190 +18,196 @@
+    .join(' ');
 }
 
 /**
- * Returns a static interactive card, you can map over arrays to populate something similar
- * @returns {Object}
+ * Returns a static interactive card, you can map over arrays to populate
+ * something similar.
+ * @return {Object}
  */
 function getInteractiveCard() {
   let imageUrls = [
-    "https://photos.app.goo.gl/5noEuKjptkjWd01n1",
-    "https://photos.app.goo.gl/DRqnKP4YBO12SnLT2"
+    'https://photos.app.goo.gl/5noEuKjptkjWd01n1',
+    'https://photos.app.goo.gl/DRqnKP4YBO12SnLT2',
   ];
   return {
     cards: [{
         header: {
-          title: "Which type",
-          subtitle: "Click on to Choose",
+          title: 'Which type',
+          subtitle: 'Click on to Choose',
           imageUrl:
-            "http://www.freepngimg.com/download/dog/9-dog-png-image-picture-download-dogs.png",
-          imageStyle: "IMAGE"
+            'http://www.freepngimg.com/download/dog/9-dog-png-image-picture-download-dogs.png',
+          imageStyle: 'IMAGE',
         },
         sections: [{
-            widgets: getWidgets(imageUrls, getImageButtonWidget)
-          }]
-      }]
+            widgets: getWidgets(imageUrls, getImageButtonWidget),
+          }],
+      }],
   };
 }
 
 /**
- * Return an array of widgets
+ * Return an array of widgets.
  * @param {Array} urls a simple array of imageUrls, strings
  * @param {Function} widgetComposer a function used to compose a widget object
- * @returns {Array}
+ * @return {Array}
  */
 function getWidgets(urls, widgetComposer) {
-  return urls.map(url => widgetComposer(url));
+  return urls.map((url) => widgetComposer(url));
 }
 
 /**
- * Returns a Widget with an image and button with a set url parameter
+ * Returns a Widget with an image and button with a set url parameter.
  * @param {String} imageUrl
- * @param {String} actionMethodName a simple string that can be passed to choose what actionMethodName is to be specified
+ * @param {String} actionMethodName a simple string that can be passed to choose
+ * what actionMethodName is to be specified
  * @return {Object} a widget with an image and button
  */
-function getImageButtonWidget(imageUrl, actionMethodName = "choose") {
+function getImageButtonWidget(imageUrl, actionMethodName = 'choose') {
   return {
     image: {
-      imageUrl
+      imageUrl,
     },
     buttons: [{
         textButton: {
-          text: "Select",
+          text: 'Select',
           onClick: {
             action: {
               actionMethodName,
               parameters: [{
-                  key: "url",
-                  value: imageUrl
-                }]
-            }
-          }
-        }
-      }]
+                  key: 'url',
+                  value: imageUrl,
+                }],
+            },
+          },
+        },
+      }],
   };
 }
 
 /**
- * Returns a Single Card with a Single Image
+ * Returns a Single Card with a Single Image.
  * @param {String} imageUrl
- * @returns {Object}
+ * @return {Object}
```

Comment:
Can you add a complete jsdoc method header?

Reply:
Not looking to improve javadoc in this PR.